### PR TITLE
Proposed changes to README files

### PR DESCRIPTION
**Pull request by rjwalters**
Draft PR for discussion purposes.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## FAQ
```
Comment (sugargoat) [inline FAQ.md:1]: Adding a FAQ on the main readme was a specific request of Brian's - so I'd like for there to at least be a reference to this file. I also think the FAQ is really helpful with honing the pitch for MobileCoin so the reader is armed with conversational-tone answers to common questions.

File: mobilecoind/README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
-## The MobileCoin Daemon
+## mobilecoind
 
-`mobilecoind` is a standalone executable, which provides blockchain synchronization and wallet services.
+The `mobilecoind` daemon is a standalone executable, which provides blockchain synchronization and wallet services.
```
Comment (carrythesky) [inline mobilecoind/README.md]: Would this be "mobilecoin daemon daemon" like when someone says "ATM machine?"

File: mobilecoind/README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+It creates encrypted, attested connections to validator nodes who are participating in federated voting in order to get the current block height, block headers, and to submit transactions. These validator nodes are considered highly trusted due to their use of SGX, and they are used as a reliable source for block information as well as to validate and process the proposed transactions from `mobilecoind`.
 
-To keep the blockchain in sync, the MobileCoin Daemon downloads new blocks from cloud storage and checks with the consensus validators that the block headers are correct. If this succeeds, they are added to a local copy of the blockchain called the Ledger DB. This is done periodically and ensures the local copy of the blockchain is fresh enough to calculate balances and generate transactions.
+To keep the blockchain in sync, the MobileCoin Daemon downloads new blocks from cloud storage and checks with the validator nodes that the block headers are correct. If this succeeds, they are added to a local copy of the blockchain called the Ledger DB. This is done periodically and ensures the local copy of the blockchain is fresh enough to calculate balances and generate transactions.
```
Comment (carrythesky) [inline mobilecoind/README.md]: if we decide to use mobilecoind, let's be consistent here? we don't introduce MobileCoin Daemon as a term above anymore.

File: mobilecoind/README.md [hunk 4]
```diff
@@ -63,9 +63,9 @@ Once you fetch the sigstruct artifact, you must provide the sigstruct to mobilec
 
 #### Example Invocation
 
-This invocation connects to two consensus validators in the MobileCoin demo network, uses their respective S3 buckets to download new blocks, polls every second for updates and provides a MobileCoinD API on port 4444.
+This invocation connects to two validator nodes in the MobileCoin demo network, uses their respective S3 buckets to download new blocks, polls every second for updates and provides a MobileCoinD API on port 4444.
 
->Note: The MobileCoin Daemon validates attestation evidence from the Consensus Validators, and so needs to know whether those validators are running with hardware SGX or in simulation mode, via the `SGX_MODE` variable. In addiiton it needs to know whether the enclave was built in debug or relase, via the `IAS_MODE` variable.
+>Note: The MobileCoin Daemon validates attestation evidence from the validator nodes, and so needs to know whether those validators are running with hardware SGX or in simulation mode, via the `SGX_MODE` variable. In addiiton it needs to know whether the enclave was built in debug or relase, via the `IAS_MODE` variable.
```
Comment (carrythesky) [inline mobilecoind/README.md:68]: ditto above

File: mobilecoind/README.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
-## The MobileCoin Daemon
+## mobilecoind
 
-`mobilecoind` is a standalone executable, which provides blockchain synchronization and wallet services.
+The MobileCoin Daemon is a standalone executable, `mobilecoind`, which provides blockchain synchronization and wallet services.
```
Comment (lucy-kind) [inline mobilecoind/README.md]: "The MobileCoin Daemon, or `mobilecoind`, is a standalone executable which provides..."

**Pull request closed without merging**